Dockerfile

```diff
@@ -18,12 +18,12 @@ RUN git clone https://github.com/containers/skopeo.git /skopeo && \
 FROM ubuntu:20.04 AS fiotools
 RUN apt-get update
 RUN apt-get install -y wget git gcc make -y
-RUN wget -P /tmp https://go.dev/dl/go1.18.linux-amd64.tar.gz && \
-    tar -C /usr/local -xzf /tmp/go1.18.linux-amd64.tar.gz
+RUN wget -P /tmp https://go.dev/dl/go1.19.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /tmp/go1.19.linux-amd64.tar.gz
```

Comment:
Better to use 1.19.8, no? (latest stable release)

Reply:
we can wait till Tuesday for 1.19.9: 
```
go1.19.9 (planned for 2023-05-02) will include security fixes to the standard library, as well as bug fixes to the compiler, the runtime, and the crypto/tls and syscall packages. 
```

Reply:
Cool, better wait for 1.19.9 then.

Reply:
@doanac mind updating the pr with .9?

Reply:
force pushed the 1.19.9 version of golang